docker/docker-compose.yaml

```diff
@@ -52,7 +52,7 @@ services:
 
   explorer-backend:
     container_name: alephium_explorer_backend
-    image: alephium/explorer-backend:v1.13.4
+    image: alephium/explorer-backend:v1.19.4
```

Comment:
you could directly go with `v2.2.1` it will be very soon the stable version

Reply:
perfect, it works too